youtube_dl/extractor/bilibili.py

```diff
@@ -155,6 +161,21 @@ def _real_extract(self, url):
                 self._report_error(js)
             cid = js['result']['cid']
 
+        subtitles = {}
+        for sub in re.findall(r'"lan":\s*"([^"]+)"[^}]+"subtitle_url":\s*"([^"]+)"', webpage, re.DOTALL):
+            subtitles[sub[0]] = [{'url': sub[1]}]
+            j = self._download_json(sub[1].replace("\\u002F", "/"), video_id, note='Downloading raw subtitle: %s' % (sub[0]))
```

Comment:
probably better:

`.j = self._download_json(sub[1].encode().decode('unicode-escape'), video_id, note='Downloading raw subtitle: %s' % (sub[0]))` to decode not only slash but also escaped characters